test/unit/specs/components/staking/ModalWithdrawAllRewards.spec.js

```diff
@@ -0,0 +1,66 @@
+"use strict"
+
+import { shallowMount } from "@vue/test-utils"
+import ModalWithdrawAllRewards from "staking/ModalWithdrawAllRewards"
+
+describe(`ModalWithdrawAllRewards`, () => {
+  let wrapper, $store
+
+  beforeEach(() => {
+    $store = {
+      commit: jest.fn(),
+      dispatch: jest.fn(),
+      getters: {
+        bondDenom: `stake`,
+        distribution: { totalRewards: { stake: 10000000 } }
+      }
+    }
+    wrapper = shallowMount(ModalWithdrawAllRewards, {
+      mocks: {
+        $store
+      },
+    })
+  })
+
+  it(`shows withdaw rewards modal`, () => {
+    expect(wrapper.vm.$el).toMatchSnapshot()
+  })
+
+  it(`form is always valid`, () => {
```

Comment:
i don't know if it makes sense for a form to always be valid

Reply:
see comment above